HelpersTask407_Rename__ck_to__csfy

Shr3yash opened the pull request:
Resolved #407 by doing the necessary modifications. Modified other uses of `is_dev_ck_example` to `is_dev_csfy_example` too.

gpsaggese reviewed (commented):
LGTM

samarth9008 commented on helpers/test/test_hserver.py:
```diff
@@ -24,7 +24,7 @@ def test_is_inside_docker1(self) -> None:
         self.assertTrue(hserver.is_inside_docker())
 
     def test_is_dev_ck1(self) -> None:
```
Need to change the test name as well.

samarth9008 reviewed (commented):
Minor change

sonniki commented on helpers/hunit_test_utils.py:
```diff
@@ -451,8 +461,8 @@ def execute_only_on_dev4() -> None:
 
 
 def execute_only_on_dev_ck() -> None:
```
Let's rename this one to `_csfy` as well (and everywhere where it's used)

sonniki commented on helpers/hunit_test_utils.py:
```diff
@@ -451,8 +461,8 @@ def execute_only_on_dev4() -> None:
-    if not is_dev_ck_:
+    is_dev_csfy_ = hserver.is_dev_csfy()
+    if not is_dev_csfy_:
         pytest.skip("Only run on dev CK")
```
`CK` -> `CSFY`

sonniki reviewed (changes requested):
A couple more

sonniki commented on helpers/test/test_repo_config_amp.py:
```diff
@@ -64,10 +64,10 @@ def test_is_dev4(self) -> None:
-        if hserver.is_dev_ck():
-            act = hserver.is_CK_S3_available()
+        if hserver.is_dev_csfy():
+            act = hserver.is_CSFY_S3_available()
```
Did you rename this function in `hserver` as well?

Shr3yash commented:
@sonniki Renamed the `is_CK_S3_available` function as per your suggestion. But now Fast and SLow tests are failing.

- `helpers/test/test_hparquet.py`
- `helpers/test/test_hs3.py`
- `helpers/test/test_hsecrets.py`

### Error Details

AttributeError: module 'helpers.hserver' has no attribute 'is_CK_S3_available'. Did you mean: 'is_AM_S3_available'?


This error indicates that the test files are referencing a non-existent attribute `is_CK_S3_available` from the `helpers.hserver` module, which should be `is_AM_S3_available`.

---

### Steps to Fix

#### 1. Update `helpers/test/test_hparquet.py`

**Original:**
```python
not hserver.is_CK_S3_available(),

Updated:

not hserver.is_AM_S3_available(),
```
2. Update helpers/test/test_hs3.py
```python
Original:

not hserver.is_CK_S3_available(),

Updated:

not hserver.is_AM_S3_available(),
```
3. Update helpers/test/test_hsecrets.py
```python
Original:

not hserver.is_CK_S3_available(),

Updated:

not hserver.is_AM_S3_available(),
```

**OR** we can revert back to `is_CK_S3_available` in the hserver.py, please tell me how to proceed around this.

sonniki commented:
To make it easier, let's revert. But my comment above was because while you have not changed the original function in `hserver`, you have changed the call to it in `test_repo_config_amp.py`. It should be either both of them changed, or none of them changed.

Shr3yash commented:
Understood, @sonniki  I renamed one of the references without changing definition. But okay, now as it looks like CI tests are searching for that particular attribute `is_CK_S3_available`  so I will revert back the change.

sonniki commented on helpers/test/test_repo_config_amp.py:
```diff
@@ -64,10 +64,10 @@ def test_is_dev4(self) -> None:
-        if hserver.is_dev_ck():
-            act = hserver.is_CK_S3_available()
+        if hserver.is_dev_csfy():
+            act = hserver.test_is_CK_S3_available()
```
There is no such function in `hserver`

sonniki commented:
@Shr3yash there is a conflict now after a recent update in `master`. Please resolve and then we'll merge.

Shr3yash commented:
> @Shr3yash there is a conflict now after a recent update in `master`. Please resolve and then we'll merge.

Done @sonniki
